### PR TITLE
fixes #21

### LINKED ISSUE #21: Deleting Boards
For deleting: `DELETE /boards/{id}`

### PR DESCRIPTION
**Pull request by javidy**
delete /boards endpoint

### CODE REVIEW

File: src/test/java/com/tasker/server/boards/rest/BoardDeletionControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RestTest
+public class BoardDeletionControllerTest {
+
+    @Resource
+    private MockMvc mockMvc;
+
+    @Test
+    public void it_returns404_not_found() throws Exception {
```
Comment (SabuhiYusif) [inline src/test/java/com/tasker/server/boards/rest/BoardDeletionControllerTest.java:23]: it returns not found if board does not exists

File: src/main/java/com/tasker/server/boards/rest/BoardDeletionController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import java.util.UUID;
+
+@RestController
+public class BoardDeletionController {
+
+    @Resource
+    BoardService boardService;
+
+    @DeleteMapping("/boards/{id}")
+    ResponseEntity<?> delete(@PathVariable String id) {
```
[the remark this answers is not in the document]
Comment (javidy) [reply src/main/java/com/tasker/server/boards/rest/BoardDeletionController.java:21]: please elaborate.. I don't know `? -> String` what this means :)

[the remark this answers is not in the document]
Comment (SabuhiYusif) [reply src/main/java/com/tasker/server/boards/rest/BoardDeletionController.java:21]: ResponseEntity<String>

[the remark this answers is not in the document]
Comment (javidy) [reply src/main/java/com/tasker/server/boards/rest/BoardDeletionController.java]: thanks , resolved

**Pull request merged**